### PR TITLE
demo mongodb atlas vision api nodejs

### PR DESCRIPTION
**Pull request by smithkent**

### CODE REVIEW

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+    (labels,landmarks and safe search)
+4.  Kick off the batch process from any web-enabled client terminal
+5.  Verify our new metadata in both the console and in our MongoDB Atlas database
+
+## Part 1: Create and configure MongoDB Atlas Free-Tier cluster on GCP
+
+1.  Create a free account on MongoDB Atlas on www.mongodb.com/cloud/atlas
+    Click on “Try Free”at the top right if you do not have an account or “Sign In” if you already have a login. 
+
+    ![image](images/image1.png)
```
Comment (jmdobry) [inline tutorials/mongodb-atlas-visionapi-nodejs/index.md]: You need to add all of your images to this Pull Request, if you look at https://github.com/GoogleCloudPlatform/community/blob/cc14586f3408559d145cd3a916c1caa9d3d16699/tutorials/mongodb-atlas-visionapi-nodejs/index.md you'll see that none of the images load.

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+
+                nano client.js
+
+                        ****************************************
+
+                        console.log('Client-side code running');
+
+                        const button = document.getElementById('myButton');
+
+                        button.addEventListener('click', function(e) {
```
Comment (jmdobry) [inline tutorials/mongodb-atlas-visionapi-nodejs/index.md]: Remove all the extra newlines, this code is hard to read, apply throughout

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+                mkdir public
+                cd public
+
+                ![image](images/image2.png)
+
+        3.  Let’s create the client side file using the following command and copying the following script content into the editor. Save the file using Ctrl + X:
+
+                nano client.js
+
+                        ****************************************
```
Comment (jmdobry) [inline tutorials/mongodb-atlas-visionapi-nodejs/index.md]: delete these asterisks, apply throughout

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+        2.  We need to create a public folder for our server and select it, by entering the following commands:
+
+                mkdir public
+                cd public
+
+                ![image](images/image2.png)
+
+        3.  Let’s create the client side file using the following command and copying the following script content into the editor. Save the file using Ctrl + X:
+
+                nano client.js
```
Comment (jmdobry) [inline tutorials/mongodb-atlas-visionapi-nodejs/index.md]: Move the filename out of the code block, apply throughout

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 5]
```diff
@@ -0,0 +1,680 @@
+---
+title: Tutorial - Google Vision API and MongoDB Atlas on Google Cloud
```
Comment (jmdobry) [inline tutorials/mongodb-atlas-visionapi-nodejs/index.md]: Delete `Tutorial - ` from the title, these are all tutorials

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md [hunk 6]
```diff
@@ -0,0 +1,680 @@
...
+description: Learn how to use Google's Vision API and MongoDB Atlas to build a metadata database with Express server and Node.js.
+author: ksmith, arajwade
+tags: Vision API, Node.js, express, MongoDB Atlas
+date_published:
```
[the remark this answers is not in the document]
Comment (ToddKopriva) [reply tutorials/mongodb-atlas-visionapi-nodejs/index.md]: I fixed this.

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md (hunk 2 above)
Comment (ToddKopriva) [reply tutorials/mongodb-atlas-visionapi-nodejs/index.md]: I cleaned the code up by removing extraneous new lines and making indentation more consistent.

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md (hunk 3 above)
Comment (ToddKopriva) [reply tutorials/mongodb-atlas-visionapi-nodejs/index.md]: done

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md (hunk 4 above)
Comment (ToddKopriva) [reply tutorials/mongodb-atlas-visionapi-nodejs/index.md]: Why?

File: tutorials/mongodb-atlas-visionapi-nodejs/index.md (hunk 1 above)
Comment (ToddKopriva) [reply tutorials/mongodb-atlas-visionapi-nodejs/index.md]: done

**Comment by ToddKopriva** [comment]
published:
https://cloud.google.com/community/tutorials/mongodb-atlas-visionapi-nodejs

**Pull request merged**